traits/trait_types.py

```diff
@@ -2593,11 +2590,13 @@ class Person(HasTraits):
     #: The default value type to use (i.e. 'constant'):
     default_value_type = DefaultValue.constant
 
-    def __init__(self, *values, **metadata):
-
-        if (len(values) == 1) and (type(values[0]) in SequenceTypes):
-            values = values[0]
-        self.values = values[:]
+    def __init__(self, values, **metadata):
+        if not isinstance(values, SequenceTypes):
+            raise ValueError(
```

Comment:
I think this is a `TypeError` (wrong sort or number of parameters) rather than a `ValueError` (right sort or number of parameters, but bad values).

Reply:
Yep, it would be consistent with the test above!

Reply:
And the check was `isinstance`, so `TypeError` makes more sense indeed.